### PR TITLE
CNV-5078: image-upload in the web console

### PR DESCRIPTION
**Pull request by ousleyp**
Jira: https://issues.redhat.com/browse/CNV-5078
Preview: https://cnv5078--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-uploading-local-disk-images-web.html

CNV 2.5/enterprise-4.6 only

@jelkosz @aglitke I welcome your reviews :)

### CODE REVIEW

File: modules/virt-uploading-image-web.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+. Click *Browse...* to open the file manager and select the image that you want to upload, or drag the file into the *Drag a file here or browse to upload* field.
+
+. Optional: Set this image as the default image for a specific operating system.
+
+.. Select the *Attach this data to a Virtual Machine operating system* check box.
+
+.. Select an operating system from the list.
+
+. Enter a name for the PVC in the *Persistent Volume Claim Name* field. This name must be unique within the project.
```
Comment (gouyang) [inline modules/virt-uploading-image-web.adoc:44]: The PVC name is filled automatically and not allowed to change based on current implementation.

Comment (ousleyp) [reply modules/virt-uploading-image-web.adoc]: Thanks for letting me know! I have changed this step to `. The *Persistent Volume Claim Name* field is automatically filled and cannot be changed. Take note of the assigned name for the PVC.`

Any other concerns? Thanks, Guohua.

**Comment by gouyang** [comment]
lgtm

File: modules/virt-uploading-image-web.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Using a compressed raw image file results in the most efficient upload.
+====
+** Compress a QCOW2 image file by using the method that is recommended for your client:
+*** If you use a Linux client, _sparsify_ the QCOW2 file by using the
+link:https://libguestfs.org/virt-sparsify.1.html[virt-sparsify] tool.
+*** If you use a Windows client, compress the QCOW2 file by using `xz` or `gzip`.
+
+.Procedure
+
+. From the side menu, click *Storage* -> *Persistent Volume Claims*.
```
Comment (bergerhoffer) [inline modules/virt-uploading-image-web.adoc:30]: I know the module is for the web console, but just wondering if you should say "From the side menu of the web console" here?

File: modules/virt-uploading-image-web.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+. Click the *Create Persistent Volume Claim* drop-down list to expand it.
+
+. Click *With Data Upload Form* to open the *Upload Data to Persistent Volume Claim* page.
+
+. Click *Browse...* to open the file manager and select the image that you want to upload, or drag the file into the *Drag a file here or browse to upload* field.
```
Comment (bergerhoffer) [inline modules/virt-uploading-image-web.adoc:36]: I think the IBM style guide recommendation is to leave out the "..." in "Browse...", even if that's what the button actually shows.

File: modules/virt-uploading-image-web.adoc [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+. Optional: Set this image as the default image for a specific operating system.
+
+.. Select the *Attach this data to a Virtual Machine operating system* check box.
+
+.. Select an operating system from the list.
+
+. The *Persistent Volume Claim Name* field is automatically filled and cannot be changed. Take note of the assigned name for the PVC.
```
Comment (bergerhoffer) [inline modules/virt-uploading-image-web.adoc:44]: Any tips as to why they need to take note of the PVC name?

**Review by bergerhoffer — COMMENTED** [review]
Just a few minor things!

File: virt/virtual_machines/virtual_disks/virt-uploading-local-disk-images-web.adoc [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+:context: virt-uploading-local-disk-images-web
+toc::[]
+
+You can upload a locally stored disk image file by using the web console.
+
+== Prerequisites
+
+* You must have a virtual machine image file in IMG, ISO, or QCOW2 format.
+
+* If you require scratch space according to the
```
[the remark this answers is not in the document]
Comment (ousleyp) [reply virt/virtual_machines/virtual_disks/virt-uploading-local-disk-images-web.adoc:13]: Whoops! I think I got halfway into setting up that link and then forgot that I needed to finish it. Good catch.

**Pull request merged**